thug/DOM/W3C/HTML/HTMLAnchorElement.py

```diff
@@ -63,6 +63,14 @@ def port(self):
 
         return self.host.split(":")[1]
 
+    @property
+    def pathname(self):
+        if not self.href:
+            return ""
+
+        o = urlparse(self.href)
+        return o.path if o.path else ""
```

Comment:
This line is not correct. According to documentation

_The HTMLAnchorElement.pathname property is a string containing an initial '/' followed by the path of the URL not including the query string or fragment (or the empty string if there is no path)._

The fix is very simple as line 72 should be

`
    return o.path if o.path else "\"
`

Can you please fix that? Thanks!

Reply:
Thank you for your review, and new code committed